app/Http/Controllers/Bible/BibleFileSetsController.php

```diff
@@ -89,80 +92,262 @@ function () use ($fileset_id, $book_id, $type, $chapter_id) {
-                $query = BibleFile::where('hash_id', $fileset->hash_id)
-                    ->leftJoin(
-                        config('database.connections.dbp.database') .
-                            '.bible_books',
-                        function ($q) use ($bible) {
-                            $q
-                                ->on(
-                                    'bible_books.book_id',
-                                    'bible_files.book_id'
-                                )
-                                ->where('bible_books.bible_id', $bible->id);
-                        }
-                    )
-                    ->leftJoin(
-                        config('database.connections.dbp.database') . '.books',
-                        'books.id',
-                        'bible_files.book_id'
-                    )
-                    ->when($chapter_id, function ($query) use ($chapter_id) {
-                        return $query->where(
-                            'bible_files.chapter_start',
-                            $chapter_id
-                        );
-                    })
-                    ->when($book, function ($query) use ($book) {
-                        return $query->where('bible_files.book_id', $book->id);
-                    })
-                    ->select([
-                        'bible_files.duration',
-                        'bible_files.hash_id',
-                        'bible_files.id',
-                        'bible_files.book_id',
-                        'bible_files.chapter_start',
-                        'bible_files.chapter_end',
-                        'bible_files.verse_start',
-                        'bible_files.verse_end',
-                        'bible_files.file_name',
-                        'bible_books.name as book_name',
-                        'books.protestant_order as book_order'
-                    ]);
-
-                if ($type === 'video_stream') {
-                    $query
-                        ->orderByRaw(
-                            "FIELD(bible_files.book_id, 'MAT', 'MRK', 'LUK', 'JHN') ASC"
-                        )
-                        ->orderBy('chapter_start', 'ASC')
-                        ->orderBy('verse_start', 'ASC');
-                }
+                
+                return $this->showAudioVideoFilesets(
+                    $bible,
+                    $fileset,
+                    $book,
+                    $chapter_id,
+                    $asset_id,
+                    $type
+                );
+            }
+        );
+
+        return $this->reply($fileset_chapters, [], $transaction_id ?? '');
+    }
+
+    /**
+     *
+     * @OA\Get(
+     *     path="bibles/filesets/{fileset_id}/{book}/{chapter}",
+     *     tags={"Bibles"},
+     *     summary="Returns Bibles Filesets",
+     *     description="Returns a list of bible filesets",
+     *     operationId="v4_bible_filesets.show",
```

Comment:
- change summary to "Returns content for a given fileset"
- change description to "For a given fileset, book and chapter, return content (text, audio or video)"
- change operationId to v4_bible_filesets.showChapter
- change book_id parameter to be in="path", required = true
- change chapter_id parameter to be in="path", required = true